deploy/helm_charts/envs/mixer_stanford.yaml

```diff
@@ -0,0 +1,25 @@
+# Helm config
+mixer:
+  gcpProjectID: datcom-stanford
+  serviceName: stanford.api.datacommons.org
+
+ingress:
+  name: mixer-ingress-standford
+  annotations:
+    {
+      kubernetes.io/ingress.global-static-ip-name: mixer-ip,
+      ingress.gcp.kubernetes.io/pre-shared-cert: mixer-certificate,
+    }
+
+kgStoreConfig:
+  customBigtable: |
+    project: datcom-stanford
+    instance: dc-graph
+    tables:
+      - private_2022_10_26_16_01_13
```

Comment:
For custom DC, do the versions live in here?

Slightly wonder if it might be more convenient (for refresh) to have it in a separate file, perhaps on GCS.

Reply:
Makes sense.